skins/shared/styles/buttons/buttons-wikia.scss

```diff
@@ -9,10 +9,10 @@ a.wikia-button,
 .wikia-single-button a,
 .wikia-menu-button,
 input[type="submit"]:not(.wds-button),
-input[type="reset"]:not(.wds-button),
-input[type="button"]:not(.wds-button),
-.button:not(.wds-button),
-button:not(.wds-button) {
+input[type="reset"],
+input[type="button"],
+.button,
+button {
```

Comment:
After that update you may also add "background-image:none;", "border-radius:0;" and "font-family:inherit;" to .wds-button class so as not to conflict with the 2010 era buttons as well. That's my proposed solution.